.github/workflows/reusable-e2e.yml

```diff
@@ -108,20 +60,40 @@ jobs:
           --slot-name ${{ inputs.slotName }} \
           --is-cicd
 
-      - name: Seed e2e database
+      - name: Seed e2e database ##If E2E CosmosDB exists and has been seeded, we do not want to seed again
```

Comment:
re-running the seed on an already seeded database risks corrupting existing data. So we check this to conditionally run the seed operation downstream